app/views/contest_ballots/index.html.erb

```diff
@@ -0,0 +1,47 @@
+<%
+  @title ||= :contest_ballot_title.t
+%>
+
+<div class="list-group-item">
+  <div>
+    <%= :contest_ballot_intro.t %><br/><br/>
+  </div>
+
+  <%= form_for(:contest_ballot, url: {action: :create}) do |form| %>
+    <div class="hidden">
+      <%= submit_tag "Submit" %>
+    </div>
+    <div><h3>
+      <%= form.check_box(:confirmed,
+                         { checked: @votes.first.confirmed,
+                           onclick: 'this.form.submit()' }) %>
+      <%= :contest_ballot_check_text.t %>
+    </h3></div>
```

Comment:
I suggest moving this to the bottom of the form, per the instructions ("When you are ready for your ballot to count, check the box at the _bottom_ of the ballot." Emphasis added.)
```suggestion
```

Reply:
Thanks for catching this inconsistency.  I actually want to revise the wording rather than move the box.  It was at the bottom and I moved it since it was not obvious.